packages/clerk-js/src/ui.retheme/elements/Form.tsx

```diff
@@ -244,10 +245,9 @@ const OTPInput = (props: OTPInputProps) => {
         {/*TODO: Create a descriptor for OTP wrapper*/}
         <Col
           elementDescriptor={descriptors.form}
-          gap={2}
+          gap={3}
+          align='center'
         >
-          <Field.OTPInputLabel />
-          <Field.OTPInputDescription />
```

Comment:
@marcelscruz Is this removed?

Reply:
It is removed indeed, it's the 2 lines of text right above the input fields on the image below.
But your comment made me realize I need to bring one of these lines to the top, and remove the current card subtitle on this page.
Thank you for the heads up.

<img width="427" alt="Screenshot 2023-12-12 at 13 49 43" src="https://github.com/clerk/javascript/assets/12413903/60ad5a13-7b8a-4638-9085-505988b9c58b">